CMSgov/mdct-mcr
# Add bss entities page; configure dynamic field array functionality

## Description
[MDCT-411](https://qmacbis.atlassian.net/browse/MDCT-411) Add BSS entities page. This adds a crucial component which is the ability for users to add and remove fields dynamically. The values from this page will later be used to have the user fill out sections of the form multiple times per answer

### How to test
- Run locally
- Navigate to `/mcpar/program-information/add-bss-entities`
- Test out page
  - Fill values of current box
  - Click to add a row, should only let you add one at a time
  - Click X to remove row - should remove the current row, not the last item added
  - Add multiple, then delete values from multiple. Click X. Should remove all blank fields
  - Should throw an error if you click _Save & continue_ while some fields are empty

## Code author checklist

## Reviewer checklist (two different people)
